webviz_subsurface/_datainput/units.py

```diff
@@ -0,0 +1,467 @@
+########################################
+#
+#  Copyright (C) 2021-     Statoil ASA
+#
+#  Webviz is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  ResInsight is distributed in the hope that it will be useful, but WITHOUT ANY
+#  WARRANTY; without even the implied warranty of MERCHANTABILITY or
+#  FITNESS FOR A PARTICULAR PURPOSE.
+#
+#  See the GNU General Public License at <http:#www.gnu.org/licenses/gpl.html>
+#  for more details.
+#
+########################################
+########################################
+#
+#  The code in this file is based on and/or inspired by opm-common,
+#  which is distributed under the GNU General Public License v3.0,
+#  and available at https://github.com/OPM/opm-common.
+#
+#  Especially in the following points this file, or files imported in this file
+#  within the same context, make use of or are based on methods developed
+#  for the opm-common project:
+#  - Reading of Eclipse INIT files
+#  - Units, unit systems and conversion of units
+#  - Class structures for storing and accessing PVT data,
+#    both base classes and derived classes for specific fluids
+#
+#  You should have received a copy of the GNU General Public License
+#  along with webviz-subsurface. If not, see (https://www.gnu.org/licenses/gpl-3.0.html).
+#
+#  (12-01-2021)
+#
+#  Copyright (C) 2009-2012 SINTEF ICT, Applied Mathematics.
+#
+########################################
+
+from typing import Union, Dict, Any
+
+
+class UnitBase:
+    """This is a pseudo-base-class necessary due to class Prefix requiring
+    a class type for operator type definitions.
+
+    TODO (RMT): Can be removed as soon as 'from __future__ import annotations'
+    is available (Python >= 3.7).
```

Comment:
Can be removed or? No annotations imported here at least?
```suggestion
```

Reply:
UnitBase is a pseudo class that is required in order to be accessible in the Prefix class which is defined prior to the Unit class. With annotations Prefix would know about the Unit class and the UnitBase class would not be required.